README.md

```diff
@@ -1,7 +1,90 @@
-# server
+# Welcome to Prospect's Backend!
 
-## npm start deploy
-  - Compiles project and deploys to production
+Hi! I'm your helpful successor who cares about knowledge transfer on this project. You will find all of the necessary links and information listed below. We will cover several topics such as: DynamoDB Table setup, AWS SDK usage, GraphQL, using Lambda to setup API's, and a few others.
 
-## npm start deploydev
-  - Compiles project and deploys to development
+# Project Structure
+
+Below are the different main folders and their contents explained.
+
+## package.json
+
+There are 4 main scripts that have been defined in the package.json file:
+
+    - "deploy": "API_ENV=production webpack && sls deploy --stage prod" -> This method deploys server functions to prod and makes sure that all tables being referenced are prod tables.
```

Comment:
These are getting rendered as a code snippet so the long lines aren't wrapping and have to be horizontally scrolled to be read. I'm thinking it might be better to make them a list and use inline code blocks if needed.